app/views/abuse_reports/new.html.erb

```diff
@@ -7,10 +7,59 @@
 <!--/subnav-->
 
 <!--main content-->
+<div class="userstuff">
+  <p><%= ts('The Abuse committee looks after all of your concerns regarding content and user behavior which breach the
+           Archive %{tos_link}. For technical help, please contact %{support_link}.',
+            tos_link: (link_to ts("Terms of Service"), tos_path), support_link: (link_to ts("Support"),
+                                                                                         new_feedback_report_path)).html_safe %></p>
+  <p><%= ts('<strong>We investigate every report we receive.</strong> For this reason, we ask you to not submit several
+           reports regarding any one issue unless you have additional information to offer. We also ask that you do not
+           encourage other users to submit a report regarding content you have already reported. Doing so slows down our
+           response and reaction time considerably, as we are a small volunteer-based committee.').html_safe %></p>
+  <p>
+    <%= ts('Please submit a report via this form if you have not submitted this report within the past fourteen (14) days,
+          and you:') %>
+  </p>
+  <ul>
+    <li><%= ts('would like to contact us regarding your <a href="http://www.archiveofourown.org/tos_faq#next_of_kin">Fannish Next of Kin</a>, or').html_safe %></li>
+    <li><%= ts('have lost or forgotten your login details, or') %></li>
+    <li><%= ts('believe you have found content uploaded to the Archive which breaches our Terms of Serivce, or') %></li>
+    <li><%= ts('would like to lodge a DMCA claim - please review our %{dmca_link}', dmca_link:
+            (link_to ts('DMCA Policy'), dmca_path)).html_safe %></li>
+  </ul>
+  <p><%= ts('<strong>We can answer Abuse reports in %{list}.</strong> Please allow for an additional delay for responses
+            in any language other than English.', list: @abuse_languages.map(&:name).to_sentence).html_safe %></p>
+</div>
 <%= form_for @abuse_report, class: "post" do |f| %>
   <fieldset>
     <legend><%= ts("Link and Describe Abuse") %></legend>
     <dl>
+      <dt><%= f.label :username, ts("Your name (optional)") %></dt>
+      <dd><%= f.text_field :username %></dd>
+      <dt class="required"><%= f.label :email, ts('Your email (required)') %></dt>
+      <dd class="required">
+        <%= f.text_field :email, size: 60, 'aria-describedby' => 'email-field-description' %>
```

Comment:
`size: 60` here and on lines 57 and 81 can go -- they're about setting the width of the field, but we use CSS for that now